addons/account/views/account_payment_view.xml

```diff
@@ -220,7 +220,7 @@
 
                             <!-- Journal Entry  button -->
                             <button name="button_open_journal_entry" type="object" class="oe_stat_button" icon="fa-bars">
-                                Journal Entry
+                                <span>Journal Entry</span>
```

Comment:
Shouldn't we had a div with `o_stat_info` as class ?

Reply:
@Ngquang ping

Reply:
@FlorianGilbert Right, i tried, and the result not really what i expected. The text in button still hidden by it's own container.

![Screenshot from 2022-07-27 10-23-22](https://user-images.githubusercontent.com/90305443/181154137-c2f87589-45fd-4fbb-bb49-d191ef994bb0.png)

But with `span` the text inside button will display with newline if there is text too long for it's own container.

![Screenshot from 2022-07-27 10-24-53](https://user-images.githubusercontent.com/90305443/181154297-e6e0e45a-bb60-4a04-b21a-f4736dca36d4.png)

Reply:
Sorry, I would mean something like that:

```html
<div class="o_stat_info">
    <span>Journal Entry</span>
</div>
```

Reply:
@FlorianGilbert I tried and the problem still there.

![Screenshot from 2022-07-27 13-36-13](https://user-images.githubusercontent.com/90305443/181178378-62522154-7739-4c15-8f97-e907a0c71c60.png)

Reply:
And just to be sure, could you please try something like that:
```html
<div class="o_stat_info">
    <span class="o_stat_text">Journal Entry</span>
</div>
```

Reply:
@FlorianGilbert Ah, it looks better. I think i will change the commit like your way.

![Screenshot from 2022-07-27 14-00-02](https://user-images.githubusercontent.com/90305443/181182669-7c1ed373-0eb4-42ab-b735-d2be8138bcee.png)